string robot_name
float64 chemical_accumulated
float64 max
float64 min
